### PR TITLE
replace invalid links with valid links to primary sources

### PR DESCRIPTION
**Pull request by finanalyst**
## The problem
Issue #2568

Modified pod files

### CODE REVIEW

File: doc/Language/classtut.pod6 [hunk 1]
```diff
@@ -314,7 +314,7 @@ initialized when being declared, however we are implementing the Singleton
 pattern here and the object must be created during its first use. It is not 100%
 possible to predict the moment when attribute initialization will be executed,
 because it can take place during compilation, runtime or both, especially when
-importing the class using the L<use|/syntax/use> keyword.
+importing the class using the C<use> keyword.
```
Comment (JJ) [inline doc/Language/classtut.pod6]: Why is this link eliminated?

File: doc/Language/nativecall.pod6 [hunk 2]
```diff
@@ -336,17 +335,17 @@ for 0..$native-array.elems - 1 -> $position {
...
+  Current element is: 1
+  Current element is: 2
+  Current element is: 3
+  Current element is: 4
+  Current element is: 5
+  Element at position 0 is 1
+  Element at position 1 is 2
+  Element at position 2 is 3
+  Element at position 3 is 4
+  Element at position 4 is 5
```
Comment (JJ) [inline doc/Language/nativecall.pod6]: As @cfa said, there's no problem with the elimination of indentation...

File: doc/Language/classtut.pod6 (hunk 1 above)
Comment (finanalyst) [reply doc/Language/classtut.pod6]: Where does it go to? I could not identify a suitable target

Comment (finanalyst) [reply doc/Language/classtut.pod6]: found appropriate place to target. Reinserted link.

**Comment by finanalyst** [comment]
All requested changes made. This PR removes 404 errors from five files

**Pull request closed without merging**